src/webviews/extension-side/dataviewer/dataViewer.node.ts

```diff
@@ -42,9 +41,10 @@ import { Telemetry } from '../../webview-side/common/constants';
 import { WebViewViewChangeEventArgs } from '../types';
 import { WebviewPanelHost } from '../webviewPanelHost.node';
 import { noop } from '../../../platform/common/utils/misc';
+import { joinPath } from '../../../platform/vscode-path/resources';
 
 const PREFERRED_VIEWGROUP = 'JupyterDataViewerPreferredViewColumn';
-const dataExplorerDir = path.join(EXTENSION_ROOT_DIR, 'out', 'webviews', 'webview-side', 'viewers');
+const dataExplorerDir = joinPath(Uri.file(EXTENSION_ROOT_DIR), 'out', 'webviews', 'webview-side', 'viewers');
```

Comment:
This should use IExtensionContext.extensionURI instead. EXTENSION_ROOT_DIR is not valid in a web context.

Reply:
Oh I guess you haven't translated this one to web yet. It doesn't really matter yet then.

Reply:
This file is still `.node.ts`